editoast/src/views/train_schedule.rs

```diff
@@ -590,7 +613,7 @@ fn build_simulation_request(
         speed_limit_tag: train_schedule.speed_limit_tag.clone(),
         power_restrictions,
         options: train_schedule.options.clone(),
-        rolling_stock: PhysicsConsistParameters::with_traction_engine(rolling_stock.into()).into(),
+        rolling_stock: physics_consist_parameters.into(),
```

Comment:
Since the only use of this parameter is to `into()` it into something else, better use the type it's converted into in the function signature for readability;